code/modules/antagonists/traitor/objectives/kidnapping.dm

```diff
@@ -77,27 +78,28 @@
 		/datum/job/lawyer,
 		/datum/job/mime,
 	)
+	alive_bonus = 1
 
 /datum/traitor_objective/kidnapping/common/assistant
 	progression_minimum = 0 MINUTES
 	progression_maximum = 15 MINUTES
+	telecrystal_reward = list(1, 2)
 	target_jobs = list(
 		/datum/job/assistant
 	)
```

Comment:
It will inherent this from the parent class, so this is unnecessary and does nothing

Reply:
yeah thats what I thought does it also inherit the bonus?

Reply:
This review got fucked up, probably because of a new commit while sending it. Ignore it.

Reply:
I removed the telecrystal reward and bonus since you told me it copies of parent is this still true?

Reply:
> I removed the telecrystal reward and bonus since you told me it copies of parent is this still true?

Yes if they are copies from the parent class remove them. I think my github is bugging as my code review is pointing at the wrong thing, but you removed the right stuff.

Reply:
cool talked to it in discord coding channel and they said the same thing

Reply:
Though the telecrystal award wasn't a duplicate. Whether you want to buff assistant objective is your choice, it was just the award_bonus that was a duplicate.

Reply:
so assitants get no initial telecrystal price? they just start at 0

Reply:
The parent class is 2, 3, so the assistant one will inherant that if not overidden. I believe you wanted assistants to stay at 1, 2, according to PR description.

Reply:
if I write the TC price on them to 1, 2 will the price go to 1,2 even if the parent si still 2,3

Reply:
Yes. When a child class defines something, it overrides its parents class definition.

Reply:
Oh and something I missed. Doesn't look like there are any other kidnap objectives that didn't override alive_bonus, so you could set alive_bonus to 1 on the base kidnap objective (var/alive_bonus = 1) and remove all the overrides of alive_bonus that is set to 1.

Reply:
I'm fine with assitants have the same worth price as common since their basically the same thing before just difference in time and rarity